src/main/java/org/variantsync/diffdetective/feature/CPPDiffLineFormulaExtractor.java

```diff
@@ -65,6 +74,13 @@ public String extractFormula(final String line) throws IllFormedAnnotationExcept
         // remove whitespace
         fm = fm.replaceAll("\\s", "");
 
+        if (fm.isEmpty()) {
+            throw couldNotExtractFormula.get();
+        }
+
+        // remove defined()
+        fm = DEFINED_PATTERN.matcher(fm).replaceAll("$1");
+        fm = fm.replaceAll("defined ", " ");
```

Comment:
Are these changes new? Did Lukas Bormann make these or you @ibbem ? Also `defined(x)` has different semantics in CPP than `x` so maybe we should not drop `defined` clauses but rather make `x` become `DEFINED_x`?

Reply:
Yes, these changes are new but there where already patches like this in #50. These are essentially duplicates which I missed during merging and should be removed. Indeed, the defined operators are just ignored since at least 9b5e59c2a2ea9a67be34e64311482e562237a401.
Adding a `DEFINED_` prefix seems good to me. I will add it soon.

Reply:
Ok good! So you removed this change for #65 ? You plan on opening a new pull-request for the DEFINED-issue in the future? Maybe opening an issue as a reminder would be good but I leave that up to you.

Reply:
Yes, this change is not included in #65. The change was already finished. See #66.